notebooks/official/pipelines/rapid_prototyping_bqml_automl.ipynb

```diff
@@ -1195,9 +1148,9 @@
         "id": "0f573556"
       },
       "source": [
-        "### Validate Infrastructure\n",
+        "### Validate the infrastructure\n",
         "\n",
-        "Once the best model has been deployed, you validate the endpoint by making a simple prediction to it."
+        "Post selecting the best model, it is deployed to an endpoint. Define a component that validates the endpoint by making prediction requests to it."
```

Comment:
- change "it is" to "it's"
- change: " Define a component that validates the endpoint by making prediction requests to it."
    -to: " Define a component that validates the endpoint by making prediction requests to that endpoint."